src/main/kotlin/kat/Config.kt

```diff
@@ -4,5 +4,5 @@ import java.lang.System.getenv
 
 object Config {
     val port = getenv("PORT")?.toInt() ?: 8080
-    val useFakeLogin = getenv("USE_FAKE_LOGIN")?.toBoolean() ?: false
```

Comment:
Is this a good idea? It's possible to configure this via Maven/IDE, why should it always be false?

Reply:
All the old tests require the "useFakeLogin" variable to be "true" during their execution. The application requires the "useFakeLogin" variable to be "false" during its execution. The newly added test also requires this variable to be "false". I set this variable to "false" by default for the application to work. I set this variable to "true" to run old tests. However, running the new test requires this variable to be "false".

Reply:
Hmm.. that's a good question, I don't actually know. I can merge it like this for now, and we can revisit it later if necessary.